### PR TITLE
Add Signup component, add clean-webpack-plugin

### PR DESCRIPTION
**Pull request by Jeremy-D**

### CODE REVIEW

File: client/src/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleClickFacebook() {}
+  handleClickGoogle() {}
+
+  handleSubmit(event) {
+    alert('A name was submitted: ' + this.state.value);
+    event.preventDefault();
+    console.log(this.state.firstName);
```
Comment (sjaymoon15) [inline client/src/Signup.jsx]: a PR should not contain any console.logs.

Comment (Jeremy-D) [reply client/src/Signup.jsx]: should i get rid of the alert as well?

Comment (sjaymoon15) [reply client/src/Signup.jsx]: yes. that would be great!

**Comment by gr2m** [comment]
should I be able to click on the SignUp button and then see a sign up?

image

right now nothing happens :)

**Comment by Jeremy-D** [comment]
I think I messed something up I'm going to trash this one and try again, sorry!

**Pull request merged**